tests/test_tranform.py

```diff
@@ -7,23 +7,25 @@
 def scan_json_datadir():
     """Returns a dict for data/json directory"""
     store = {}
-    json_dir = 'data/json/'
+    json_dir = os.path.abspath('tests/data/json/')
     json_files = os.listdir(json_dir)
     for file in json_files:
-        f = open(json_dir + f'{file}')
-        data = f.read()
+        file_dir = os.path.join(json_dir, file)
```

Comment:
I suggest renaming file_dir to file_path

Reply:
Updated